sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1,5871 +1,6462 @@
+                        "eq": ""
+                    },
+                    "allowCalculateOverride": false,
+                    "showCharCount": false,
+                    "showWordCount": false,
+                    "allowMultipleMasks": false,
+                    "addons": [],
+                    "inputType": "hidden",
+                    "id": "ez16imf"
+                },
+                {
+                    "label": "CIP code",
+                    "labelPosition": "top",
+                    "labelWidth": "",
+                    "labelMargin": "",
+                    "placeholder": "",
+                    "description": "Format (##.####)",
+                    "tooltip": "",
+                    "prefix": "",
+                    "suffix": "",
+                    "widget": {
+                        "type": "input"
+                    },
+                    "inputMask": "",
+                    "displayMask": "",
+                    "allowMultipleMasks": false,
+                    "customClass": "",
+                    "tabindex": "",
+                    "autocomplete": "",
+                    "hidden": false,
+                    "hideLabel": false,
+                    "showWordCount": false,
+                    "showCharCount": false,
+                    "mask": false,
+                    "autofocus": false,
+                    "spellcheck": true,
+                    "disabled": false,
+                    "tableView": true,
+                    "modalEdit": false,
+                    "multiple": false,
+                    "persistent": true,
+                    "inputFormat": "plain",
+                    "protected": false,
+                    "dbIndex": false,
+                    "case": "",
+                    "truncateMultipleSpaces": false,
+                    "encrypted": false,
+                    "redrawOn": "",
+                    "clearOnHide": true,
+                    "customDefaultValue": "",
+                    "calculateValue": "",
+                    "calculateServer": true,
+                    "allowCalculateOverride": false,
+                    "validateOn": "change",
+                    "validate": {
+                        "required": true,
+                        "minLength": "",
+                        "maxLength": "",
+                        "minWords": "",
+                        "maxWords": "",
+                        "pattern": "",
+                        "customMessage": "",
+                        "custom": "valid = (/^[0-9]{2}\\.[a-zA-Z]{4}$/.test(input) && [\"11\", \"09\", \"52\",\"01\",\"03\",\"05\", \"16\", \"23\", \"24\", \"25\", \"42\",\"27\", \"30\", \"38\", \"40\", \"45\", \"50\", \"54\", \"55\",\"13\", \"19\", \"28\", \"31\", \"44\", \"60\", \"04\", \"10\", \"14\", \"15\", \"29\", \"41\", \"26\", \"22\", \"43\", \"51\", \"39\", \"12\", \"21\", \"46\", \"47\", \"48\"].includes(input.substring(0, 2))) ? true : 'Incorrect Format';",
```

Comment:
As we talked please test if the below regex can handle the entire validation.
```regex
^(11|09|12){1}\.[a-zA-Z]{4}$
```

Reply:
I can confirm it is working as expected. I have added the other digits and updated the validation.